lib/wallaby/query.ex

```diff
@@ -289,4 +295,8 @@ defmodule Wallaby.Query do
         |> Keyword.put_new(:maximum, opts[:maximum])
     end
   end
+
+  defp add_get(opts) do
+    Keyword.put_new(opts, :get, nil)
```

Comment:
Do we always have to add it? Ah I guess that's the pattern the code above follows so - good for following the established patterns! :+1: